core/src/main/java/bisq/core/app/CoreModule.java

```diff
@@ -70,8 +72,10 @@ protected void configure() {
 
         bind(File.class).annotatedWith(named(STORAGE_DIR)).toInstance(config.storageDir);
 
-        CoinFormatter btcFormatter = new ImmutableCoinFormatter(config.networkParameters.getMonetaryFormat());
-        bind(CoinFormatter.class).annotatedWith(named(FormattingUtils.BTC_FORMATTER_KEY)).toInstance(btcFormatter);
+        String baseCurrencyNetworkCurrencyCode = config.baseCurrencyNetwork.getCurrencyCode();
+        MonetaryFormat monetaryFormat = new MonetaryFormat().noCode().code(0, baseCurrencyNetworkCurrencyCode);
+        CoinFormatter xmrFormatter = new ImmutableCoinFormatter(monetaryFormat);
```

Comment:
Why does `ImmutableCoinFormatter(config.networkParameters.getMonetaryFormat());` not work ?

Reply:
because it has "BTC" hardcoded in bitcoinj library: org.bitcoinj.params.AbstractBitcoinNetParams#getMonetaryFormat

config.networkParameters are from BaseCurrencyNetwork and there it is initialized with RegTestParams.get() which has the monetary format with BTC: org.bitcoinj.params.AbstractBitcoinNetParams#getMonetaryFormat